### PR TITLE
feat: remove useless actions from instance toolbar

### LINKED ISSUE #1148: Proposed UI Changes
### Role

I use the polymc user interface often

### Suggestion

Group several of the UI features together

### Benefit

makes the UI easier to read, and improves flow

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

I suggest that in every part of the user interface for the PolyMC launcher, the "Edit Instance", "Edit Notes", "View Mods", "View Worlds", and "Manage Screenshots" buttons be merged into one button, since they all lead to the same console window. This would, in my opinion, make using the launcher less intimidating for new users, improve "workflow" for average users, and increase the aesthetic beauty of the interface. In addition, I suggest that the "Minecraft Folder", "Config Folder", and "Instance Folder" buttons be grouped into one single button as well. This would serve the same purpose as the aforementioned changes suggested, as all three buttons open your computer's file manager to a specific path. Seeing as all three paths are less than one folder apart from each other, I find having all three as separate buttons tedious and unnecessary. Thank you for reading my suggestion, and feel free to discuss the proposed changes at will.

### PR DESCRIPTION
**Pull request by DioEgizio**
Fixes https://github.com/PolyMC/PolyMC/issues/1148

removes a few redundant buttons from the instance toolbar

### CODE REVIEW

**Comment by ryanccn** [comment]
why not remove the "instance"s in the buttons

it's an instance sidebar after all, what would it do except change stuff related to the selected instance

**Comment by Scrumplex** [comment]
~~Moving our UI changes to 6.0~~

**Review by ryanccn — APPROVED** [review]
:>

File: launcher/ui/MainWindow.cpp [hunk 1]
```diff
@@ -687,7 +687,7 @@ class MainWindow::Ui
 
         actionEditInstance = TranslatedAction(MainWindow);
         actionEditInstance->setObjectName(QStringLiteral("actionEditInstance"));
-        actionEditInstance.setTextId(QT_TRANSLATE_NOOP("MainWindow", "Edit Inst&ance..."));
+        actionEditInstance.setTextId(QT_TRANSLATE_NOOP("MainWindow", "Edit."));
```
Comment (ryanccn) [inline launcher/ui/MainWindow.cpp]: Edit.

**Review by ryanccn — CHANGES_REQUESTED** [review]

**Review by ryanccn — APPROVED** [review]
:> 2

**Review by LennyMcLennington — CHANGES_REQUESTED** [review]
Why is this removing the shortcuts for some of the menu entries? The ...s being removed is fine but I don't think removing the shortcuts makes sense.

**Comment by DioEgizio** [comment]
I just removed the "instance" part

**Comment by LennyMcLennington** [comment]
> I just removed the "instance" part

Including the ampersand shortcuts.

File: launcher/ui/MainWindow.cpp [hunk 2]
```diff
@@ -741,20 +741,20 @@ class MainWindow::Ui
 
         actionViewSelectedInstFolder = TranslatedAction(MainWindow);
         actionViewSelectedInstFolder->setObjectName(QStringLiteral("actionViewSelectedInstFolder"));
-        actionViewSelectedInstFolder.setTextId(QT_TRANSLATE_NOOP("MainWindow", "&Instance Folder"));
+        actionViewSelectedInstFolder.setTextId(QT_TRANSLATE_NOOP("MainWindow", "Folder"));
```
Comment (flowln) [inline launcher/ui/MainWindow.cpp]:
```suggestion
        actionViewSelectedInstFolder.setTextId(QT_TRANSLATE_NOOP("MainWindow", "&Folder"));
```
also here (not sure if the F is already taken though) :p

File: launcher/ui/MainWindow.cpp [hunk 3]
```diff
@@ -797,20 +797,10 @@ class MainWindow::Ui
...
-        instanceToolBar->addAction(actionEditInstNotes);
-        instanceToolBar->addAction(actionMods);
-        instanceToolBar->addAction(actionWorlds);
-        instanceToolBar->addAction(actionScreenshots);
         instanceToolBar->addAction(actionChangeInstGroup);
 
-        instanceToolBar->addSeparator();
-
-        instanceToolBar->addAction(actionViewSelectedMCFolder);
-        instanceToolBar->addAction(actionConfig_Folder);
```
Comment (Scrumplex) [inline launcher/ui/MainWindow.cpp:809]: make sure that you delete all `action`s that are unused now.

**Comment by forkiesassds** [comment]
This seems to fix #1148

File: launcher/ui/MainWindow.cpp [hunk 4]
```diff
@@ -687,74 +667,34 @@ class MainWindow::Ui
...
-        actionScreenshots = TranslatedAction(MainWindow);
-        actionScreenshots->setObjectName(QStringLiteral("actionScreenshots"));
-        actionScreenshots.setTextId(QT_TRANSLATE_NOOP("MainWindow", "Manage &Screenshots"));
-        actionScreenshots.setTooltipId(QT_TRANSLATE_NOOP("MainWindow", "View and upload screenshots for this instance."));
-        all_actions.append(&actionScreenshots);
-
         actionChangeInstGroup = TranslatedAction(MainWindow);
         actionChangeInstGroup->setObjectName(QStringLiteral("actionChangeInstGroup"));
-        actionChangeInstGroup.setTextId(QT_TRANSLATE_NOOP("MainWindow", "&Change Group..."));
+        actionChangeInstGroup.setTextId(QT_TRANSLATE_NOOP("MainWindow", "&Change Group"));
```
Comment (Scrumplex) [inline launcher/ui/MainWindow.cpp]: `...` was actually fine, as that implies that a dialog will open.

**Comment by BeansBeefBroccoli** [comment]
> This seems to fix #1148

Thats what the first message says

> Fixes https://github.com/PolyMC/PolyMC/issues/1148
> 
> removes a few redundant buttons from the instance toolbar

**Comment by DioEgizio** [comment]
> > This seems to fix #1148
> 
> Thats what the first message says
> 
> > Fixes #1148
> > removes a few redundant buttons from the instance toolbar

... Because I edited it after that comment

**Review by Scrumplex — APPROVED** [review]

**Pull request closed without merging**